packages/client/lib/client/index.ts

```diff
@@ -279,6 +279,9 @@ export default class RedisClient<
   #monitorCallback?: MonitorCallback<TYPE_MAPPING>;
   private _self = this;
   private _commandOptions?: CommandOptions<TYPE_MAPPING>;
+  #dirtyWatch?: string;
+  #epoch: number;
```

Comment:
does it default to Zero? don't see it set besides the ++ on 'ready'

Reply:
wondering if the epoch should be moved into the RedisSocket and not in the client itself.

Reply:
100% agree with you

Reply:
fixed